definitions/divorce/json/CaseEventToComplexTypes/CaseEventToComplexTypes-pet-sol-selects-own-org-nonprod.json

```diff
@@ -1,35 +0,0 @@
-[
-  {
-    "LiveFrom": "04/02/2021",
```

Comment:
moved to:
https://github.com/hmcts/div-ccd-definitions/pull/707/files#diff-83796fb8f385898c0176819d792b7360aa1f33cd7ad7f3de707ce00a8cad26e0R67-R99

It's going to be released together as a RRJ.